rust-toolchain.toml

```diff
@@ -0,0 +1,12 @@
+[toolchain]
+channel = "stable"
```

Comment:
We need rustfmt from nightly. Dont think there is a good way to specify that though...

Reply:
Ugh, that seems non-trivial indeed.

Reply:
[Rust toolchains](https://rust-lang.github.io/rustup/overrides.html) don't allow to mix component versions, I'm afraid (this isn't nix-specific). It's either all nightly or all stable.

Reply:
We might be okay there, since we always have to run `cargo +nightly fmt` anyway